models/core/validators.sql

```diff
@@ -0,0 +1,84 @@
+{{
+	config(
+		materialized='incremental',
+		unique_key='log_id',
+		tags=['core'],
+		cluster_by=['day_date']
+	)
+}}
+
+with delegators_incremental as (
+	select *
+	from {{ ref('stg_delegators') }}
+	where {{ incremental_load_filter('ingest_timestamp') }}
+),
+
+latest_day_timestamps as (
+    select
+        max(ingest_timestamp) as ingest_timestamp,
+        validator_address
+    from delegators_incremental
+    group by day_date, validator_address
```

Comment:
@chuxinh hey, is there a way to do group by 1,2 if you don't actually select the columns?

Reply:
nope i don't think `day_date` is needed here
have you tried running dbt on this? it will tell u it if goes through

Reply:
day_date is needed here because we're trying to take the latest daily snapshot..

Reply:
@eevui :D.. add day_date, change to group by 1,2

Reply:
Should I add `day_date` to the select? I only wanted to treat this CTE as a "primary key" of sorts and `day_date` would be repeated unnecessarily.

Reply:
@antonyip pushed a couple commits addressing this